app/styles/ui/_diff.scss

```diff
@@ -286,17 +286,36 @@
   }
 }
 
-.image-diff-two-up {
+.image-diff-container {
   display: flex;
   flex: 1;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
 
-  .image-diff-previous {
-    margin-right: 7.5px;
+  .image-diff-two-up {
+    display: flex;
+    align-items: center;
+
+    .image-diff-previous {
+      margin-right: 7.5px;
+      margin-bottom: 7.5px;
+    }
+    .image-diff-current {
+      margin-left: 7.5px;
+      margin-bottom: 7.5px;
+    }
```

Comment:
I think lines 304-307 can be removed if you take go forward with my suggestions above.

Reply:
Unfortunately this won't work, see https://github.com/desktop/desktop/pull/4403#discussion_r187312119